logic/createGoogleCalendar.js

```diff
@@ -5,7 +5,7 @@ let respond;
 showLogs = true; //true = shows lots of console log stuff, false shows "ERROR containing messages only"
 function logthis(stuff) {
   if (showLogs || stuff.includes("ERROR")) {
-    console.log(stuff);
+    logthis(stuff);
```

Comment:
this is an infinite loop

Reply:
What is holding up my merges? This is fixed..